components/sections/Buy.js

```diff
@@ -0,0 +1,130 @@
+import Link from "next/link";
+
+
+export default function Wallets() {
+
+    const title = "How to get Koin";
+    const subtle = "You can receive it from your peers, or buy it from exchanges ";
+
+    const wallets = [
+        {
+            name: "MEXC",
+            // description: "this is a description",
+            icon: "/images/buy/round/mexc-logo.png",
+            chrome: false,
+            android: false,
+            ios: false,
+            url: "https://www.mexc.com/"
```

Comment:
For all of the exchange links, I'd probably link directly to their KOIN listing instead of their main landing page

Reply:
If people connect from the US they will get an error message, at least on MEXC, which is not a good look. 

<img width="1840" alt="image" src="https://github.com/koinos/koinos-io-website/assets/70003/aeb10527-41c0-456e-9de7-39cfde2a1d2c">

Reply:
ok yeah, makes sense